.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+<!-- Thanks for filing an OSCC issue! You're making our world a better place!-->
+
+<!-- Questions about purchasing and pricing for OSCC boards or DriveKit?
+Shoot an email to drivekit@polysync.io -->
+
+
+
+<!--
+Interested in a new feature?
+-->
+
+### Proposed feature
+<!-- What would you like to accomplish? -->
+
+### Use case
+<!-- How would you like to do it? -->
+
+
+
+<!--
+Missing documentaion?
+-->
+
+### Expected Information
+<!-- What information would help you? -->
+
+### Available Documentation
+<!-- What were you able to find?-->
```

Comment:
Tiniest comment ever - no space between the end of the sentence and the `--`. This comment is unbelievably trivial, feel free to ignore.